lib/db_connection/connection.ex

```diff
@@ -1,5 +1,17 @@
 defmodule DBConnection.ConnectionError do
-  defexception [:message, severity: :error]
+  defexception [:message, severity: :error, metadata: []]
```

Comment:
Let's go with:

```suggestion
  defexception [:message, severity: :error, reason: :error]
```

WDYT?